.circleci/config.yml

```diff
@@ -128,6 +130,17 @@ jobs:
           command: |
             ./tools/third-party-licenses-check.sh
 
+  build:
+    docker:
+      - image: circleci/python:3.7
+
+    steps:
+      - checkout
+      - run: git submodule update --init --recursive
+      - setup_remote_docker
+      - run: docker build -t drafter .
+      - run: echo '# My API' | docker run -i drafter
```

Comment:
This is a primitive check to make sure that the image is capable of parsing the document. For example, all the dynamic libraries that Drafter depends on are available. If it fails it will exit 1 and fail the CI.